llarp/router/router.cpp

```diff
@@ -109,6 +110,18 @@ namespace llarp
     _linkManager.PersistSessionUntil(remote, until);
   }
 
+  void
+  Router::GossipRCIfNeeded(const RouterContact rc)
+  {
+    /// if we are not a service node forget about gossip
+    if(not IsServiceNode())
+      return;
+    /// wait for random uptime
+    if(Uptime() < _randomStartDelay)
+      return;
+    _rcGossiper.GossipRC(rc);
```

Comment:
Unless I'm missing something it seems like this is going to gossip to the network a lot.  We really want to rate limit this to once/hour since the last successful gossip, or something like once per five minutes if the last successful gossip was > an hour ago.

Reply:
Never mind, there's a filter check inside `GossipRC()` that prevents this.